features/admin/remove_services.feature

```diff
@@ -0,0 +1,28 @@
+@remove-a-service
+Feature: Remove a service
+
+Scenario: Only Service Manager can remove services
+  Given I am logged in as the production admin-ccs-category user
+  # TODO GET ME HERE FROM THE LOGIN
+  And I am on the /admin/services/1123456789012346 page
```

Comment:
This is another one where we shouldn't rely on a particular service already existing - should set up all the data we need for the test in a "Backgound" step if possible.